CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
+# 3.0.1
```

Comment:
`# Unreleased`

Reply:
It was this that made me believe I could add the new version immediately, and create a release just after merging this PR:
<img width="999" alt="Screenshot 2021-10-15 at 10 57 29" src="https://user-images.githubusercontent.com/8872303/137461698-326af6f1-d01b-43e4-a1d6-845ab8be26d5.png">

Reply:
Hmm... forgot that one... I guess yes, but not certain if it's a good practice or not. It's less overhead to create a release branch right away, so my gut feeling is that it's it a good idea :)